task/jq/0.1/tests/pipelinerun.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: new-pipeline
+spec:
+  pipelineRef:
+    name: new-pipeline
+  taskRunTemplate:
+    serviceAccountName: pipeline
```

Comment:
this serviceaccount is specific to openshift, can you please remove this?

Reply:
Done!